platform/linux-generic/include/odp_crypto_internal.h

```diff
@@ -40,8 +42,10 @@ struct odp_crypto_generic_session {
 	odp_bool_t do_cipher_first;
 
 	struct {
+#if ODP_DEPRECATED_API
 		/* Copy of session IV data */
 		uint8_t iv_data[MAX_IV_LEN];
+#endif
```

Comment:
Needing an #ifdef looks ugly here and elsewhere in this series. Better to correct the code than retain "compatibility" here.